packages/api/package.json

```diff
@@ -30,7 +30,7 @@
   "main": "index.js",
   "modules": "index.js",
   "dependencies": {
-    "autobahn": "18.10.2",
+    "autobahn": "19.4.1",
```

Comment:
Bumping autobahn right before a release. What got into you ;-)

Reply:
Hope I wont regret that indeed 😬